# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#feedback
int8 	progress				# Progress of the scanning process
int16 	state					# STOPPED=0, RECEIVED=1, STARTED=2, FINISHED=3, FAILED=-1
time 	scan_request_received	# timestamp for when the primitive was recieved at the primitive executor..

